src/components/login/SelectAccountDropdown.js

```diff
@@ -101,14 +115,11 @@ const SelectAccountDropdown = ({
                 basic
                 onClick={!disabled ? handleOnClick : () => { }}
                 className={disabled && 'disabled'}
-                title={!disabled 
-                    ? translate('selectAccountDropdown.switchAccount') 
-                    : translate('selectAccountDropdown.switchAccounthNotAllowed')
-                }
+                title={!dropdown ? (!disabled ? translate('selectAccountDropdown.switchAccount') : translate('selectAccountDropdown.switchAccounthNotAllowed')) : ''}
```

Comment:
@Patrick1904 nit: let's try to avoid using one ternary operator expression inside another, it's somewhat confusing, especially as one liner.